tap_github/repository_streams.py

```diff
@@ -92,6 +92,20 @@ def query(self) -> str:
                     )
                 return "query {" + " ".join(chunks) + " rateLimit { cost } }"
 
+            def validate_response(self, response: requests.Response) -> None:
```

Comment:
This looks ok. Though you could also imagine doing a try/except around your new graphql implementation. Wdyt?